docs/browser/events.md

```diff
@@ -0,0 +1,50 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Events
+--->
+
+# Semantic conventions for browser events
+
+**Status**: [Development][DocumentStatus]
+
+This document defines semantic conventions for browser (web) instrumentations
+that emit events.
+
+## WebVital Event
+
+<!-- semconv event.browser.web_vital -->
+<!-- NOTE: THIS TEXT IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/snippet.md.j2 -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-capture -->
+<!-- markdownlint-disable -->
+
+**Status:** ![Development](https://img.shields.io/badge/-development-blue)
+
+The event name MUST be `browser.web_vital`.
+
+This event describes the website performance metrics introduced by Google, See [web vitals](https://web.dev/vitals).
```

Comment:
this is an interesting example of something we would probably generally define as an otel metric

is the idea that otel metrics will almost never make sense in a browser, and we just want the ability to emit individual metric measurements from browser instrumentation?

Reply:
let's split this out into follow-up work, I'll open a tracking issue